server/mailet/rate-limiter/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+# In order to start James JPA app on top of mariaDB:
```

Comment:
Bad copy paste spotted! It's james memory here, nothing to do with the JPA version :)